docs/registry/quarkus-registry/README.adoc

```diff
@@ -194,35 +193,33 @@ endif::[]
 == Creating the quotes Kafka topic in {product-kafka}
 
 [role="_abstract"]
-For this quick start, the Kafka topic that the Quarkus example application uses is called `quotes`. You must create this topic in {product-kafka} so that the Quarkus application can interact with it.
+The Quarkus application in this quick start uses a Kafka topic called `quotes` to produce and consume messages. In this task, you'll create the `quotes` topic in your Kafka instance.
 
 .Prerequisites
-* You're logged in to the {product-kafka} web console at {service-url-kafka}[^].
-* You have a running Kafka instance with at least one Kafka topic in {product-kafka} (see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^]).
+* You have a running Kafka instance in {product-long-kafka}.
```

Comment:
Given that you're creating the required `quotes` topic in this task, I don't think it's the case that you need a Kafka instance that already has at least one topic.